atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -59,7 +59,7 @@ def run(self):
 
         while True:
             # Pull the image from Crane
-            name = self.tasker.pull_image(pullspec)
+            name = self.tasker.pull_image(pullspec, insecure=True)
```

Comment:
Agree, a config option is a good idea. Which default value should we stick to - I guess `insecure=False`?